uber/automated_emails.py

```diff
@@ -12,34 +12,69 @@ class AutomatedEmail:
         Attendee: lambda session: session.staffers(only_staffing=False),
         Group: lambda session: session.query(Group).options(subqueryload(Group.attendees))
     }
-    extra_models = queries  # we've renamed "extra_models" to "queries" but are temporarily keeping the old name for backwards-compatibility
 
-    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
+    def __init__(self, model, subject, template, filter, *, date_filters=[], sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
```

Comment:
It's considered better to say ``date_filters=(),`` in a function parameter list.  The reason is that this can happen:

```python
def add_many(count, item, xs=[]):
    for i in range(count):
        xs.append(item)
    return xs

add_many(2, 'apple')  # returns ['apple', 'apple']
add_many(2, 'grape')  # returns ['apple', 'apple', 'grape', 'grape']
```

Do you see what's happening?  The default parameter is initialized once when the function is defined, and then ever after that the same object is used.  Therefore if you want an iterable, it's considered better to make the default an immutable version of that iterable, e.g. a tuple instead of a list or a frozenset instead of a set.

In this case it probably doesn't matter, but seeing this immediately made me nervous, so it's probably worth changing.